invoke-ai/InvokeAI
# fix: workflows backcompat

## What type of PR is this? (check all applicable)


## Description

Old workflows appear to "just work" (with one caveat*). I _really_ did not expect this. To be honest, I don't really understand how it doesn't explode.

Model related fields were the big question mark, but what actually ends up happening is they show up with an error state, and force you to choose a model.

The only change in this PR is to fix the styling for some field types.

*Caveat: If your workflow was already loaded in the editor and you updating the app, it model fields seem to just not render at all. If you load the workflow after updating the app, the thing I don't understand happens and it works. I maybe able to work that mystery mojo into the app startup but maybe this is good enough?

## QA Instructions, Screenshots, Recordings

Load up workflows saved on a previous version. The worst that should happen is you have to re-select models.



## Merge Plan

This PR can be merged when approved
